src/java/src/main/java/com/dnanexus/DXProject.java

```diff
@@ -219,6 +223,458 @@ public void destroy(boolean terminateJobs) {
                 RetryStrategy.SAFE_TO_RETRY);
     }
 
+    /**
+     * Move specified files or folder to an archive storage to save on storage costs.
+     *
+     * <p>
+     * Example use:
+     * </p>
+     *
+     * <pre>
+     * // Archive using file id
+     * DXFile f1 = DXFile.getInstance(&quot;file-xxxx&quot;);
+     * DXFile f2 = DXFile.getInstance(&quot;file-yyyy&quot;);
+     * ArchiveResults r = project.archive().addFiles(f1, f2).execute();
+     * // Archive folder
+     * ArchiveResults r = project.archive().setFolder(&quot;/folder&quot;, true).execute();
+     * </pre>
+     *
+     * @return a newly initialized {@code ArchiveRequestBuilder}
+     */
+    public ArchiveRequestBuilder archive() {
+        return new ArchiveRequestBuilder();
+    }
+
+    /**
+     * Retrieve specified files or folder from an archive storage.
```

Comment:
Can you reword this to "Request specified files or folder to be unarchived"

Reply:
done